cloudhub-2/modules/ROOT/pages/ch2-ha-dr.adoc

```diff
@@ -0,0 +1,210 @@
+
+==== Application State
+
+* Set up the infrastructure across multiple regions. In the passive region, you can deploy your backup applications either in a fully scaled-up state or in a scaled-down state, with fewer replicas or smaller replica sizes. Scaled-down applications increase reliance on the control plane to redeploy and scale up during an outage in the primary runtime plane region.
+* You can configure CPU-based Horizontal Pod Autoscaling (HPA) where applicable if available to your organization. For more information, see xref:ch2-configure-horizontal-autoscaling.adoc[].
+
+==== Traffic Management
+
+* Use a BYO global load balancer to route traffic to only one region at a time.
+* Perform health checks to determine DR and switch routing.
+* For non-HTTP use cases, such as Schedulers, externalize a DR flag in your application and use a Groovy script to enable and disable scheduler flows directly in the runtime plane. This reduces reliance on the control plane to trigger your backup applications via a cold standby approach.
+
+==== Licensing
+
+* For organizations using UBP, this configuration doesn’t consume messages or throughput. The configuration consumes fewer flows if you configure the application with xref:ch2-configure-horizontal-autoscaling.adoc[CPU-based HPA].
+* For organizations not using UBP, this configuration consumes lower vCores if the backup deployment is in a scaled-down state before it scales up in response to an outage in the primary region.
+
+==== Recovery Time Objective
+
+If the backup application is in a fully scaled state, you experience lower disruption in the event of an outage. If the backup application is in a scaled-down state, the RTO depends on the time it takes to scale the application up either through control-plane-triggered redeployments, or through xref:ch2-configure-horizontal-autoscaling.adoc[CPU-based HPA].
+
+For integrating systems to resend events or data of in-flight transactions, capture recovery points in logs and monitoring. This builds reliability in your applications.
+
+=== Use Case 3: Cold Standby Configuration
+
+==== When to Use
+
+Low-impact, stateless, or stateful applications that tolerate longer downtime during recovery.
+
+==== Application State
+
+* Set up the infrastructure, but configure your application to a stopped state in the passive region.
+* Start the application only if you detect an outage.
+
+==== Traffic Management
+
+* Use a BYO global load balancer to route traffic only to the region that is up.
+* Perform health checks to determine DR, and scripts to start your applications in backup state and switch traffic routing.
+* If the control plane is unavailable, you can't start the backup application in the runtime plane.
+
+==== Licensing
+
+Because the backup application is in a stopped state, this configuration doesn't consume additional vCores, Mule flows, Mule messages or Data throughput until the backup application starts in the event of a disaster.
```

Comment:
```suggestion
Because the backup application is in a stopped state, this configuration doesn't consume additional vCores, Mule flows, Mule messages, or Data throughput until the backup application starts in the event of a disaster.
```